build.bat

```diff
@@ -368,6 +430,62 @@ if %BUILD_JAVA% == 1 (
 
 )
 
+::------------------------------------------------------------------------------
+if !BUILD_MICRO! == 1 (
+
+	call::solution_compilation_flag_calculation
+
+	if !BUILD_MICRO_24x_COMPATIBILITY! == 1 (
+		set "ADDITIONAL_DEFINES=RTI_MICRO_24x_COMPATIBILITY !ADDITIONAL_DEFINES!"
+	) else (
+		set "rtiddsgen_extra_options=!rtiddsgen_extra_options! -sequenceSize !MICRO_UNBOUNDED_SEQUENCE_SIZE! -additionalRtiLibraries nddsmetp"
+	)
+
+	set "ADDITIONAL_DEFINES=RTI_WIN32 RTI_MICRO !ADDITIONAL_DEFINES!"
+	set "additional_header_files=MessagingIF.h RTIDDSImpl.h perftest_cpp.h CpuMonitor.h PerftestTransport.h Infrastructure_common.h Infrastructure_micro.h"
+	set "additional_source_files=RTIDDSImpl.cxx CpuMonitor.cxx PerftestTransport.cxx Infrastructure_common.cxx Infrastructure_micro.cxx"
+
+	@REM # Generate files for srcCpp
+	echo[
+	echo [INFO]: Generating types and makefiles for %classic_cpp_lang_string%
+	call "%rtiddsgen_executable%" -micro -language %classic_cpp_lang_string% -replace^
+	-create typefiles -create makefiles^
+	-additionalHeaderFiles "!additional_header_files!"^
+	-additionalSourceFiles "!additional_source_files!" -additionalDefines "!ADDITIONAL_DEFINES!"^
+	!rtiddsgen_extra_options!^
+	-d "%classic_cpp_folder%" "%idl_location%\perftest.idl"
+	if not !ERRORLEVEL! == 0 (
+		echo [ERROR]: Failure generating code for %classic_cpp_lang_string%.
+		exit /b 1
+	)
+	call copy "%classic_cpp_folder%"\perftest_publisher.cxx "%classic_cpp_folder%"\perftest_subscriber.cxx
+	call type NULL > "%classic_cpp_folder%"\perftestApplication.h
+	call type NULL > "%classic_cpp_folder%"\perftestApplication.cxx
+	call copy "%idl_location%\perftest.idl" "%classic_cpp_folder%"\perftest.idl
+
+	echo[
+	echo [INFO]: Compiling %classic_cpp_lang_string%
+
+	cd "%classic_cpp_folder%"
+	call !CMAKE_EXE! -DCMAKE_BUILD_TYPE=!RELEASE_DEBUG! --target perftest_publisher -G %cmake_generator% -B./perftest_build -H. -DRTIME_TARGET_NAME=%architecture% -DPLATFORM_LIBS="netapi32.lib;advapi32.lib;user32.lib;winmm.lib;WS2_32.lib;"
+	if not !ERRORLEVEL! == 0 (
+		echo [ERROR]: Failure compiling code for %classic_cpp_lang_string%.
+		cd ..
+		exit /b 1
+	)
+	call !CMAKE_EXE! --build ./perftest_build --config !RELEASE_DEBUG!
+	if not !ERRORLEVEL! == 0 (
+		echo [ERROR]: Failure compiling code for %classic_cpp_lang_string%.
+		cd ..
+		exit /b 1
+	)
+	cd ..
+
+	echo [INFO]: Copying perftest_cpp executable file:
+	md "%bin_folder%"\%architecture%\!RELEASE_DEBUG!
+	copy /Y "%classic_cpp_folder%"\objs\%architecture%\!RELEASE_DEBUG!\perftest_publisher.exe "%bin_folder%"\%architecture%\!RELEASE_DEBUG!\perftest_cpp_micro.exe
```

Comment:
At this point this path does not exist:
`"%classic_cpp_folder%"\objs\%architecture%\!RELEASE_DEBUG!\perftest_publisher.exe `

We are not creating the folder `!RELEASE_DEBUG!`
This should be `"%classic_cpp_folder%"\objs\%architecture%\perftest_publisher.exe`
Or create the folder `!RELEASE_DEBUG!`